modules/rodoo-uninstall-resources.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-uninstall.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-uninstall-resources_{context}"]
+= Uninstalling {run-once-operator} resources
+
+Optionally, after uninstalling the {run-once-operator}, you can remove its related resources from your cluster.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* You have uninstalled the {run-once-operator}.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Remove CRDs that were created when the {run-once-operator} was installed:
+.. Navigate to *Administration* -> *CustomResourceDefinitions*.
+.. Enter `RunOnceDurationOverride` in the *Name* field to filter the CRDs.
+.. Click the Options menu {kebab} next to the *RunOnceDurationOverride* CRD and select *Delete Custom Resource Definition*.
+.. In the confirmation dialog, click *Delete*.
+
+. Delete the `openshift-run-once-duration-override-operator` namespace.
+.. Navigate to *Administration* -> *Namespaces*.
+.. Enter `openshift-run-once-duration-override-operator` into the filter box.
+.. Click the Options menu {kebab} next to the *openshift-run-once-duration-override-operator* entry and select *Delete Namespace*.
+.. In the confirmation dialog, enter `openshift-run-once-duration-override-operator` and click *Delete*.
+
+// TODO: Do we care about unlabeling namespaces?
```

Comment:
@ingvagabund should we worry about telling users that they can remove the label for RODOO from the namespaces that they added it to? wdyt?

Reply:
We should. Better to increase awareness rather than wait until a customer reports the labeling can be undone for user-created namespace. This is unfortunately something the operator can not control.